### PR TITLE
Remove special charachters from eRPM field

### PR DESCRIPTION
**Pull request by tbolin**
The labels of the RPM fields was changed from `eRPM(/100)` to `eRPM` in 
https://github.com/betaflight/betaflight/pull/13123 . This PR does the same in log viewer to so the RPM fields are once again correctly formatted.

### CODE REVIEW

**Review by ledvinap — APPROVED** [review]

**Comment by haslinghuis** [comment]
Will try again build later (most times these are network errors)

image

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Will try again build later (most times these are network errors)

image

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
